Cargo.toml

```
@@ -245,12 +245,13 @@ codegen-units = 1
 inherits = "dev"
 debug-assertions = false
 
+[profile.bench]
+lto = true
+
 # Enable debug information specifically for memory profiling.
 # https://docs.rs/dhat/0.2.1/dhat/#configuration
-#
-# 2021-01-08: This would be nicer as a named profile, e.g. [profile.memory]
-# https://github.com/rust-lang/cargo/issues/6988
-[profile.bench]
+[profile.memory]
+inherits = "release"
 lto = true
 debug = true
 debug-assertions = false
```

The `bench` profile inherits from `release` ([ref](https://doc.rust-lang.org/cargo/reference/profiles.html#bench)). This means you can make this inherit from `bench` and skip the LTO key. We also don't need the debug-assertions key as that's already false in `release`/`bench`.

```suggestion
inherits = "bench"
debug = true
```

---

Made it inherit from bench.

I left the debug-assertions line for clarity since it was already there.

---

When I read this I had to go confirm that the bench profile in fact does not include debug assertions. It's also unexplained why bench-memory has this and bench doesn't.

---

Ok